AdvancedRocketry/blocks/Hatches.php

```diff
@@ -1,45 +1,62 @@
-    <p>Item Hatches are used to provide a way of getting items into and out of a machine.  Most will move an itemstack over if there is room if the output stack cannot fit in the last slot.</p>
-    <p>Energy Hatches are used to accept or output energy from a machine.</p>
-    <p>Hatches can be linked directly to a rocket, a rocket assembler, or Docking Pad using a <a href="../items/linker.php">Linking tool</a>.</p>
-    <p>When linked to a Docking Pad or rocket assembler, any rocket that lands on the Docking Pad or on the landing pad will automatically be linked to the hatches.</p>';
-    
+
+    //USE <p></p> to make a new paragraph
+    $description1 = '
+        <p>Hatches are a way of inputting and outputting items, liquid, and energy to/from machines. A single hatch can
+            be made as either an Input or Output hatch.  As an example, most machines will require and Power Input Hatch
+            to receive power, an Input Hatch for input items, and an Ouput Hatch for returning the finished products.
+            However, machines which handle fluids (like the <a href="/ChemicalReactor.php">Chemical Reactor</p>), may
+            require both Fluid Input Hatches for input fluids (e.g., water), and Fluid Output Hatches for output fluids
+            (e.g., Hydrogen and Oxygen).</p>
+        <p>Input Hatches have a few inventory slots for holding items to be fed into the machine. As the machine
+            operates these will be consumed. The results of the operation are then placed in Output Hatches. Either
+            running out of inputs, or filling out the outputs will cause the machine to pause.</p>
+        <p>Fluid Input Hatches have an internal reservior to hold a small amount of fluid, and can be filled either by
+            placing a bucket in the slot, or right-clicking with a filled bucket.  They can also accept fluids from
+            other mods\' pipe sysems (e.g., BuildCraft or Thermal Dyanamics). Fluid Output Hatches work in much the same
+            way.  Most machines will stop operating with input tanks are empty, or output tanks are full.</p>
+        <p>Most machines require a Power Input Plug to receive power from some kind of power grid. This can either be
+            a Power Ouput Hatch from some a power-producting machine from Advanced Rocketry (e.g., the
+            <a href="MicrowaveReceiver.php">Microwave Receiver</a>), any kind of generator from another mod, or a power
+            conduit (e.g., a Fluxduct from Thermal Dynamics). Power Ouput Plugs are included in machines which
+            generate power (like the Microwave Receiver previously mentioned) so that the power it generates may be
+            transmitted to other machines or a power grid.</p>
+        <p id="rockets">There are also a set of hatch-like blocks which are specialized for loading and unloading
```

Comment:
I left this here because there are at least a few other pages which refer people to Hatches instead of Loaders/Unloaders.